### ISSUE TITLE
nx-maven:run-task skipProject option

### ISSUE THREAD

**Issue by gitLinda** [khalilou88/jnxplus#986]
Hi

I am using the `nx-maven:run-task` executor to deploy maven atifacts. 
Now I have the usecase that I sometimes want to skip the deploy. I saw you have this `skipProject` option. But I'm not fully understanding it. 

```
if (!options.skipProject) {
    command += ` -pl :${context.projectName}`;
}
```

If I call it with `skipProject = true`, no `-pl` flag will be added, so it would run for everything.

How is the `skipProject` option supposed to be used?

**Comment by khalilou88** [comment]
The idea is to not specify a project for a generic command especially for root project or to give users the ability to use a relative path as project identifier (-pl,--projects : Comma-delimited list of specified reactor projects to build instead of all projects. A project can be specified by [groupId]:artifactId or by its relative path).

```
      "release" : {
         "executor": "@jnxplus/nx-maven:run-task",
         "options" : {
            "skipProject": "true",
            "task": "deploy -Drevision=${VERSION} -DskipTests=true -pl ../{projectRoot}"
         }
      },
```

Maybe this option is not so useful.

**Comment by gitLinda** [comment]
I see, would it be possible to add a generic `skip` option?

**Comment by khalilou88** [comment]
Can you please describe your use case?

**Comment by khalilou88** [comment]
I feel you want to mix `nx:noop` with `nx-maven:run-task` or maybe for unit tests.

**Comment by gitLinda** [comment]
We are using code generators to set up repositories where we also generate the targets. Sometimes people want to release images and sometimes they want to publish JARs. I would like to generate both targets for them, but skip it the maven deploy by default. If they want it, they can enable it.

**Comment by khalilou88** [comment]
@gitLinda Okay i added `skipExecutor` option here: https://github.com/khalilou88/jnxplus/pull/996

**Comment by gitLinda** [comment]
Great! Thanks :)

**Comment by khalilou88** [comment]
I published version v0.45.0 for `skipExecutor` option
